src/main/java/com/todolist/app/service/ToDoStoreSingleton.java

```diff
@@ -32,10 +41,11 @@ public String getUuid() {
 
     /**
      * This method adds a new task to todo list
-     * @param s - it's a task, which must be added
+     * @param item - it's a task, which must be added
      */
-    public void addItem(String s){
-        items.put(getUuid(), new Item(s, Status.INCOMPLETED));
+    public void addItem(Item item){
+        items.put(getUuid(), item);
+        logger.info("New task was added!");
```

Comment:
Let's add UUID of task and text into logging so that we are able to identify the full flow in the logs if necessary.

Because just `New task was added!` isn't well informative. In case of issue occurred it will be impossible to identify the root cause.